src/views/auths/2FaForm.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+import { useNavigate } from 'react-router-dom';
+import OTPinput from '../../components/formControlscomponents/2FAinput/OtpInputt';
+// eslint-disable-next-line import/no-unresolved
+import Button from '../../components/formControlscomponents/Button/Button';
+
+export const URL = `https://ecommerce-app-legends-bn-production.up.railway.app`;
+// eslint-disable-next-line react/function-component-definition
+const TwoFaForm = () => {
+  const [otpFields, setOtpfileds] = useState(['', '', '', '', '', '']);
+  const [activeField, setActiveField] = useState(0);
+  const [verifyButton, setverifyButton] = useState({
+    display: 'hidden',
+    text: `verify OTP`,
+  });
+  const inputRef = useRef(null);
+  const navigate = useNavigate();
+  //  handle change input field changing events
+  const handleChange = (e, index) => {
+    const { value } = e.target;
+    const newOTP = [...otpFields];
+    newOTP[index] = value.substring(value.length - 1);
+    setOtpfileds(newOTP);
+    // checking whether there is a value in field
+    if (!value) setActiveField(index - 1);
+    else setActiveField(index + 1);
+  };
+  //  handle focus every time input needed to the next field
+  useEffect(() => {
+    inputRef.current?.focus();
+  }, [activeField]);
+  // handle submition of OTP for verification
+  const handleVerify = async () => {
+    // eslint-disable-next-line no-alert
+    setverifyButton({
+      display: 'inline-block',
+      text: `Loading...`,
+    });
+    const OTP = otpFields.join('');
```

Comment:
Use descriptive variable names - OTP is a commonly-used acronym in technology, but not everyone may understand it. Consider using a more descriptive variable name, such as verificationCode, to make it easier for others to understand what the variable represents.